src/domains/event/repository/event.repository.impl.ts

```diff
@@ -116,6 +116,44 @@ export class EventRepositoryImpl implements EventRepository {
     return new TimeTrackingDTO(event);
   }
 
+  /**
+   * Retrieves blocker events related to a specific issue.
+   * @param issueId The unique identifier of the issue.
+   * @returns An array of blocker status modification DTOs.
```

Comment:
For future PR, please, use types like this with JSdoc:
```suggestion
   * @returns {Promise<BlockerStatusModification[]>} A promise that resolves to array of blocker status 
   * modification DTOs.
```